postinstall

```diff
@@ -1,20 +1,20 @@
 #!/bin/bash
 
-return_code=0
+[ "${BASH_SOURCE[0]}" != "$0" ] && { echo "ERROR: this script must be executed by Bash, neither source'd nor executed by a different shell"; return 1; }
+
+set -e
 if [[ -n "${BROWSER_DRIVER_INSTALLER_CHROME_VERSION:-}" && -n "${BROWSER_DRIVER_INSTALLER_CHROMEDRIVER_PATH:-}" ]]; then
     node ./index.js --browser-name Chrome                                        \
                     --browser-version "$BROWSER_DRIVER_INSTALLER_CHROME_VERSION" \
-                    --target-path "$BROWSER_DRIVER_INSTALLER_CHROMEDRIVER_PATH" || return_code=$?
-else 
-    echo "The environment variables for Chrome are not set, skipping post-install for Chrome"
+                    --target-path "$BROWSER_DRIVER_INSTALLER_CHROMEDRIVER_PATH"
+else
+    echo "The environment variables for Chrome (\$BROWSER_DRIVER_INSTALLER_CHROME_VERSION, \$BROWSER_DRIVER_INSTALLER_CHROMEDRIVER_PATH) are not set, skipping the post-install step for Chrome"
```

Comment:
remove the $ (it's not part of the env var name)

Reply:
fixed